app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/AccountCheckController.scala

```diff
@@ -49,4 +49,4 @@ class AccountCheckController @Inject()(
       }
   }
 
```

Comment:
For the case here with multipleAccounts the result should now redirect to the new endpoint you've just created rather than return OK("multiple Accounts with Nino")